kube/resources/scheduler.yaml

```diff
@@ -51,7 +61,7 @@ spec:
             - name: WAIT_BEFORE_HOSTS
               value: "5"
             - name: WAIT_HOSTS
-              value: airbyte-db-svc:5432
+              value: $(DATABASE_HOST):$(DATABASE_PORT)
```

Comment:
I don't think this works? 

```
→ kubectl describe pod airbyte-scheduler-7764b66c8c-5n6bm | grep WAIT_HOSTS\:
      WAIT_HOSTS:               $(DATABASE_HOST):$(DATABASE_PORT)
```

We should probably remove wait entirely since we're waiting within code with finer control.